gbp/scripts/pq.py

```diff
@@ -487,7 +487,8 @@ def main(argv):
         except GitRepositoryError:
             # Not being on any branch is OK with --ignore-branch
             if options.ignore_branch:
-                current = repo.get_commits(num=1,
+                current = repo.get_commits(
+                    num=1,
```

Comment:
The "branch name" is o.k. but did you verify e.g. "gbp switch" works with this? It would be good to have a test in `tests/component/deb/test_pq.py` that makes sure we don't break this behavior. Most of the functions called assume it's a branch we're working on.